db/seed.sql

```diff
@@ -38,25 +38,25 @@ VALUES ("Dragon Con", "The larget multi-media popculture convetion focusing on s
 
 
 --INTEREST TABLE data
-INSERT INTO interest (event_name, event_desc)
+INSERT INTO interest (interest_name, event_desc)
```

Comment:
This section is still showing as "event_desc" - it should be "interest_desc" based on the schema.sql